packages/tools/package/jest.config.js

```diff
@@ -0,0 +1,3 @@
+const { modern } = require('../../../config/jest');
```

Comment:
Question: So this jest.config.js now pulls the configuration from the config folder and applies that globally to jest commands being run on all packages inside webex, calling etc, unless there is another jest.config.js within a package. Is my understanding right?

Reply:
This config file is specifically applied to the `@webex/package-tools` package and its scope. It does not run tests against any packages like the `@webex/legacy-tools` does.

* `@webex/legacy-tools` - Provides tooling to legacy packages ( `./packages/@webex/**` ) for building, testing, etc
* `@webex/package-tools` - Provides CI/CD/automation (and soon downstream tooling) tooling to the root of the package, github actions, and soon to be downstream consumers for organizing and managing package versions and deployments.